pact-core/Pact/Core/IR/Eval/CEK.hs

```diff
@@ -161,8 +161,13 @@ evaluateTerm cont handler env (Var n info)  = do
           returnCEKValue cont handler (VClosure (CT clo))
         Just d ->
           throwExecutionError info (InvalidDefKind (defKind d) "in var position")
+          -- TODO ^ shall this be an invariant failure?
+          -- apparently this can never happen because `renameTerm` checks for variable top-level variable occurrences,
+          -- erroring out with `InvalidDefInTermVariable` if it's violated.
+          -- And, the execution can't change that invariant once it's established during desugar.
         Nothing ->
           throwExecutionError info (NameNotInScope (FullyQualifiedName mname (_nName n) mh))
+          -- TODO ^ ditto, `renameTerm` apparently always fails in this case as well
```

Comment:
yes, but I want info about the name not in scope

Reply:
Maybe just the show instance is fine? Or just `renderQualifiedName` or something